devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy

```diff
@@ -87,7 +87,10 @@ def parse(String description) {
             // manufacturer-specific attribute
             event = getPressureResult(Integer.parseInt(descMap.value, 16))
         } else if (descMap?.clusterInt == zigbee.LEVEL_CONTROL_CLUSTER && descMap.attrInt == 0x0000) {
-            if (Integer.parseInt(descMap.value, 16) == 255) state.obstructed = true
+            if (Integer.parseInt(descMap.value, 16) == 255) {
```

Comment:
Are you sure that a value of 255 is supposed to mean it is obstructed? Mine is reporting 255 after join even though it's not obstructed.

Reply:
It might be similar to the garage door opener where it's unsure of it's actual position until it has been actuated?

this 255 is cribbed directly from the previous DTH